Origin: http://websvn.kde.org/?view=revision&revision=1268881
Description: This patch prevents a kscd crash when this program is started
 without having a disc in the device.
Applied-Upstream: 4.8.0
--- a/kscd/kscd.cpp
+++ b/kscd/kscd.cpp
@@ -69,7 +69,9 @@ KSCD::KSCD( QWidget *parent ) : KscdWind
 
 	/** Music Brainz initialisation	*/
 	m_MBManager = new MBManager();
-	m_MBManager->discLookup(devices->getMedia()->currentSource().deviceName());
+	if (devices->getMedia()) {
+		m_MBManager->discLookup(devices->getMedia()->currentSource().deviceName());
+	}
 
 	setupActions();
 	setupContextMenu();
